__tests__/api.test.js

```diff
@@ -120,4 +120,49 @@ describe("GET, Reviews",()=>{
             expect(body.msg).toBe("page not found")
         })
     })
-})
\ No newline at end of file
+})
+
+describe("GET, Recieved list of comments by Review ID",()=>{
+    test("200: recieved list of comments",()=>{
+        return request(app)
+            .get("/api/reviews/2/comments")
+            .expect(200)
+            .then(({body}) => {
+                expect(body.comments).toHaveLength(3)
+                body.comments.forEach(commentEntry => {
+                expect(commentEntry).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    review_id: 2
+                    })
+                })
+            })
+    })
+    test("200: Recieve empty array",()=>{
```

Comment:
Just be careful to keep these descriptive enough - 'Receive an empty array for a review with no comments'. Great job including it, though, as this is the one that's often forgotten!